courses/free-courses-en.md

```diff
@@ -606,6 +607,11 @@
 * [What is Deep Learning](https://www.udacity.com/course/deep-learning--ud730) (Udacity)
 
 
+### Designing
+
+* [Design 101 Crash Course: Learn UX/UI Design, Figma](https://youtu.be/cKZEgtQUxlU?si=cFyZKITD_X7aiQj-) - Daniel Schifano (ZTM)
```

Comment:
this is a shortened link - we don't allow those. 
For youtube, we prefer "playlist" links. docs/CONTRIBUTING.md#platforms-and-access-notes
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. I like `UX/UI` better as a category. Odd that we don't have more resources for this!